src/index.js

```diff
@@ -5,14 +5,14 @@ import App from "./App";
 import { store } from "./app/store";
 import { Provider } from "react-redux";
 import * as serviceWorker from "./serviceWorker";
-import { StyledEngineProvider } from "@mui/material/styles";
+import { StyledEngineProvider } from '@mui/material/styles';
 
 ReactDOM.render(
-  <StyledEngineProvider injectFirst>
     <Provider store={store}>
+      <StyledEngineProvider injectFirst>
       <App />
-    </Provider>
-    </StyledEngineProvider>,
+      </StyledEngineProvider>
+    </Provider>,
```

Comment:
Is there should be a "," after </Provider> or is it a typo? Not sure but well done :)

Reply:
Line no 15 "," is compulsory, but I can put en extra as well but it's optional not needed.